recipes/hmntrimmer/run_test.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+echo -e "\n\n*** TEST ***\n\n"
+
+HmnTrimmer 2> /dev/null || [[ "$?" == 1 ]]
+make test
```

Comment:
Typically we [don't recommend running unit tests](https://bioconda.github.io/contributor/guidelines.html#tests) due to the risk of timing out CI infrastructure, but it seems to be fine here. Just FYI.

Reply:
Got it